tests/test_gil_scoped.py

```diff
@@ -97,3 +97,9 @@ def test_python_to_cpp_to_python_from_process():
 def test_cross_module_gil():
     """Makes sure that the GIL can be acquired by another module from a GIL-released state."""
     m.test_cross_module_gil()  # Should not raise a SIGSEGV
+
+
+# Checks for segfault
```

Comment:
# Attempts to generate non-deterministic Python 3.9 MacOS segfault.
?

Reply:
Yes, though I probably will just pull this one experimental test; having our tests fail due to this, now that 3.9 is out, is probably not a good idea as long as we are actively trying to solve the issue. And this isn't breaking the tests anyway...